assets/map.js

```diff
@@ -162,9 +166,19 @@ function initMap() {
 var displayMarkers = function(infoWindow) {
 
     // Generate markers
-    var markers
-    $.get("https://raw.githubusercontent.com/archisgore/dive-site-markers/master/markers.json").done(function(cm) {
-        markers = JSON.parse(cm);
+    var markers;
+    $.get(MARKER_URL).done(function(cm) {
+        console.log(typeof cm)
+        try {
+            markers = JSON.parse(cm);
+        } catch (e) {
+            if (typeof cm === "object") {
```

Comment:
I could also just use cm.toString() I suppose.

Reply:
I think this should read - so it reads like a cascade:
```
if (typeof cm == "string") {
    markers = JSON.parse(cm); //can have a try-catch to console.write JSON parsing errors.
} else if (typeof cm == "object") {
    markers = cm;
} else {
    console.log("Unknown object type: " + typeof(cm));
}
```